pkg/agent/endpoints/workload/handler.go

```diff
@@ -149,18 +162,24 @@ func (h *Handler) attest(pid int32) []*common.Selector {
 		}
 	}
 
+	h.T.AddSampleWithLabels([]string{"workload_api", "discovered_selectors"}, float32(len(selectors)), tLabels)
 	return selectors
 }
 
 // invokeAttestor invokes attestation against the supplied plugin. Should be called from a goroutine.
 func (h *Handler) invokeAttestor(a workloadattestor.WorkloadAttestor, pid int32, sChan chan []*common.Selector, errChan chan error) {
+	attestorName := h.attestorName(a)
+	tLabels := []telemetry.Label{{"workload_pid", string(pid)}, {"attestor_name", attestorName}}
+
 	req := &workloadattestor.AttestRequest{
 		Pid: pid,
 	}
 
+	start := time.Now()
 	resp, err := a.Attest(req)
+	h.T.MeasureSinceWithLabels([]string{"workload_api", "attestor_latency"}, start, tLabels)
```

Comment:
attest_duration and attest_latency seems very similar

Reply:
attest_duration captures how long an individual PID takes to go through the entire attestation process, which is (should be?) the max(attest_latency) by pid. I figured there are two perspectives here: 1) tell me how long it's taking my workload to pass attestation, and 2) tell me which attestors take the longest. You _should_ be able to derive the former from the latter but I figured metrics are cheap and maybe it's better to just measure both... I can remove attest_duration though if you think it's more noise than value.

Reply:
@walmav is this OK? Would you prefer if I change the latter to `attestor_latency`?

Reply:
Yes this is fine. We could keep them both.